src/main/java/seedu/address/model/event/EventNameContainsKeywordsPredicate.java

```diff
@@ -0,0 +1,46 @@
+package seedu.address.model.event;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+import seedu.address.commons.util.StringUtil;
+import seedu.address.commons.util.ToStringBuilder;
+
+/**
+ * Tests that a {@code Volunteer}'s {@code Name} matches any of the keywords given.
+ */
+public class EventNameContainsKeywordsPredicate implements Predicate<Event> {
```

Comment:
ouh shit i did the exact same thing as u HAHAHAH telepathy?

Reply:
HAHAHAH